# Hi-M protocol

*Hi-M: Direct and simultaneous observation of transcription and chromosome architecture in single-cells*

This repository contains a 'static' version of our Hi-M protocol on August 9, 2019.

Live versions and updates to the protocol will be published regularly on this Repository.

If you want to contribute, please do so by creating a pull request. When doing, ensure that your changes are incremental, otherwise revision of large number of changes in difficult to manage.

#### More about Hi-M

The original Hi-M method was published [here](https://www.cell.com/molecular-cell/fulltext/S1097-2765(19)30011-5). You can access the full text at [BioRxiv](https://www.biorxiv.org/content/10.1101/434266v2).

From this original development, we wrote a full descriptive protocol. This ‘static’ protocol has now been accepted in Nature Protocols. The link will appear after publication. See title and abstract below.

To continue updating this protocol, we decided to make it into a live Repository that (hopefully) will always be up to date.



#### Static Hi-M protocol

**Direct and simultaneous observation of transcription and chromosome architecture in single cells with Hi-M**

*Simultaneous observation of 3D chromatin organization and transcription at the single cell level and with high spatial resolution may hold the key to unveil the mechanisms regulating embryonic development, cell differentiation and even disease. We have recently developed a novel technology, Hi-M, that allows for the sequential labeling, 3D imaging and localization of multiple genomic DNA loci together with RNA expression in single cells within whole, intact Drosophila embryos. Importantly, Hi-M enables simultaneous detection of RNA status and chromosome organization without sample unmounting and probe re-hybridization. Here, we provide a step-by-step protocol describing the design of probes, the preparation of samples, the stable immobilization of embryos into microfluidics chambers, and the complete procedure for image acquisition. The combined RNA/DNA FISH procedure takes 4-5 days including embryo collection. In addition, we describe image analysis software to segment nuclei, detect genomic spots, correct for drift and produce Hi-M matrices. A typical Hi-M experiment takes 1-2 days to complete all rounds of labeling and imaging and 4 additional days for image analysis. All these procedures can be accomplished by a competent graduate student or experienced technician. This technology can be easily expanded to investigate cell differentiation in cultured cells, or organization of chromatin within complex tissues.*

